src/renderer/viz/expressions/classification/GlobalEqIntervals.js

```diff
@@ -39,7 +41,12 @@ export default class GlobalEqIntervals extends Classifier {
     }
 
     _updateBreakpointsWith (metadata) {
-        const { min, max } = metadata.stats(this.input.name);
+        if (this.input.propertyName === CLUSTER_FEATURE_COUNT) {
+            throw new CartoValidationError(`${cvt.INCORRECT_TYPE} 'clusterCount' can not be used in GlobalEqIntervals. Use ViewportEqIntervals instead`);
```

Comment:
I would change a bit the message to:
```suggestion
            throw new CartoValidationError(`${cvt.INCORRECT_TYPE} 'clusterCount' can not be used in GlobalEqIntervals. Consider using ViewportEqIntervals instead`);
```